sql-plugin/src/main/spark320/scala/com/nvidia/spark/rapids/shims/ScanExecShims.scala

```diff
@@ -41,7 +41,15 @@ object ScanExecShims {
         (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.STRUCT + TypeSig.MAP +
           TypeSig.ARRAY + TypeSig.BINARY + TypeSig.DECIMAL_128).nested(),
         TypeSig.all),
-      (fsse, conf, p, r) => new FileSourceScanExecMeta(fsse, conf, p, r)),
+      (fsse, conf, p, r) => {
+        // TODO: HybridScan supports DataSourceV2
+        // TODO: HybridScan only supports Spark 32X for now.
```

Comment:
What happens if you try to enable it on anything else? Is the error message clear?

Reply:
For DataSourceV2 checks, will add a check.
For Spark versions: Refer to comment: https://github.com/NVIDIA/spark-rapids/pull/11720/#discussion_r1877097062